pallets/gear/src/benchmarking/tests/syscalls_integrity.rs

```diff
@@ -1352,3 +1389,95 @@ where
     }
     .into()
 }
+
+fn free_range_wasm<T: Config>() -> WasmModule<T>
```

Comment:
Part of this WASM can be inside `alloc_free_wasm` so `check_mem` and `check_free_range` can be merged